premerge/premerge_resources/main.tf

```diff
@@ -19,7 +19,7 @@ resource "kubernetes_namespace" "llvm_premerge_controller" {
 
 resource "kubernetes_namespace" "llvm_premerge_linux_runners" {
   metadata {
-    name = "llvm-premerge-linux-runners"
+    name = var.linux_runners_namespace_name
   }
 }
 
```

Comment:
Again, why are we not doing this for the libcxx runners?

Reply:
The libcxx runners don't benefit really at all from caching, so they don't need any setup for this.